Makefile

```diff
@@ -194,6 +197,32 @@ release: clean $(GOPATH)/bin/gh-release cross ## Create release and upload to Gi
 	gh-release checksums sha256
 	gh-release create minishift/minishift $(MINISHIFT_VERSION) master v$(MINISHIFT_VERSION)
 
+.PHONY: release_systemtray## Works only in mac environment, export GITHUB_USER, GITHUB_TOKEN
```

Comment:
@anjannath I am little confused by your comment here. Do you mean I can not run `make release_systemtray` on a Linux system?

Reply:
@anjannath Never mind, it is fine. I got confused with `make  ci_release_systemtray`.